samples/constitutions/default/actions.js

```diff
@@ -1494,4 +1494,28 @@ const actions = new Map([
       function (args) {},
     ),
   ],
+  [
+    "remove_old_jwt_tables",
+    new Action(
+      function (args) {
+        // Validate that no arguments are passed
+        if (
+          args !== undefined &&
+          args !== null &&
+          Object.keys(args).length > 0
+        ) {
+          throw new Error(
+            "remove_old_jwt_tables does not accept any arguments",
+          );
+        }
```

Comment:
We have a helper function that does this already - best to use that for a consistent error message.

```suggestion
        checkNone(args);
```